<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>512</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:rgb(239, 244, 209);}</string>
   </property>
   <widget class="QWidget" name="Container" native="true">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>32</y>
      <width>286</width>
      <height>449</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#Container{
background-color:#1D1E4E;
border:10px solid #1D1E4E;
border-radius: 25px;
}</string>
    </property>
    <widget class="QLabel" name="Login_text">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>10</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Open Sans;
font-style: normal;
font-weight: 800;
font-size: 41px;
line-height: 56px;
/* identical to box height */


color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Signup">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>320</y>
       <width>122</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #00FF85;
border-radius: 20px;
font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 14px;
line-height: 19px;
color: #1D1E4E;
</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_email">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>105</y>
       <width>211</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#input_email{
	font: 12pt &quot;Open Sans&quot;;
background: #FFFFFF;
border-radius: 19px;
font-family: Open Sans;
font-weight: normal;
font-size: 14px;
line-height: 39px;

color: #000000;
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_confirm">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>176</y>
       <width>211</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#input_confirm{
font: 12pt &quot;Open Sans&quot;;
background: #FFFFFF;
border-radius: 19px;
font-family: Open Sans;
font-weight: normal;
font-size: 14px;
line-height: 39px;

color: #000000;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="error">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>293</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Open Sans&quot;;
color:red;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Open Sans;
font-style: italic;
font-weight: normal;
font-size: 15px;
line-height: 39px;

color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>152</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Open Sans;
font-style: italic;
font-weight: normal;
font-size: 15px;
line-height: 39px;

color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_password">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>247</y>
       <width>211</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#input_password{
font: 12pt &quot;Open Sans&quot;;
background: #FFFFFF;
border-radius: 19px;
font-family: Open Sans;
font-weight: normal;
font-size: 14px;
line-height: 39px;

color: #000000;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>223</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Open Sans;
font-style: italic;
font-weight: normal;
font-size: 15px;
line-height: 39px;

color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>Confirmation Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>365</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Open Sans;
font-style: italic;
font-weight: bold;
font-size: 10px;
line-height: 27px;
text-align: center;

color: #FFE606;</string>
     </property>
     <property name="text">
      <string>- OR -</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>405</y>
       <width>20</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#ffffff;
border-radius:10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>405</y>
       <width>20</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../google-logo-9824 2.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>381</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Open Sans;
font-style: italic;
font-weight: normal;
font-size: 10px;
line-height: 27px;
text-align: center;

color: #FFE606;</string>
     </property>
     <property name="text">
      <string>Sign in with</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Container1">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>105</y>
       <width>252</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#Container1{
	font: 12pt &quot;Open Sans&quot;;
background: #FFFFFF;
border-radius: 19px;
font-family: Open Sans;
font-weight: normal;
font-size: 14px;
line-height: 39px;

color: #000000;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Container2">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>176</y>
       <width>252</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#Container2{
	font: 12pt &quot;Open Sans&quot;;
background: #FFFFFF;
border-radius: 19px;
font-family: Open Sans;
font-weight: normal;
font-size: 14px;
line-height: 39px;

color: #000000;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Container3">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>247</y>
       <width>252</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#Container3{
	font: 12pt &quot;Open Sans&quot;;
background: #FFFFFF;
border-radius: 19px;
font-family: Open Sans;
font-weight: normal;
font-size: 14px;
line-height: 39px;

color: #000000;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>Container3</zorder>
    <zorder>Container2</zorder>
    <zorder>Container1</zorder>
    <zorder>Login_text</zorder>
    <zorder>Signup</zorder>
    <zorder>error</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_5</zorder>
    <zorder>input_confirm</zorder>
    <zorder>input_password</zorder>
    <zorder>input_email</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
